Framework/API/src/ExperimentInfo.cpp

```diff
@@ -491,16 +491,12 @@ void ExperimentInfo::setNumberOfDetectorGroups(const size_t count) const {
 void ExperimentInfo::setDetectorGrouping(
```

Comment:
This method is public, but as far as I can tell it is not used externally. It's called by `cacheDetectorGroupings`, which is only used by `ConvertToMD`, basically I'm wondering if the try-catch is needed anymore.

Reply:
It strips out any bad detector IDs and we still have many places that set bad IDs so this cannot be removed.

Reply:
Ok. I think that is probably true. So lets leave this alone.